packages/forklift-console-plugin/locales/en/plugin__forklift-console-plugin.json

```diff
@@ -101,17 +101,17 @@
   "Edit VDDK init image": "Edit VDDK init image",
   "Empty": "Empty",
   "Endpoint": "Endpoint",
-  "Ensure the URL includes the \"/ovirt-engine/api\" path. For example: https://rhv-host-example.com/ovirt-engine/api": "Ensure the URL includes the \"/ovirt-engine/api\" path. For example: https://rhv-host-example.com/ovirt-engine/api",
-  "Ensure the URL includes the \"/sdk\" path. For example: https://vCenter-host-example.com/sdk": "Ensure the URL includes the \"/sdk\" path. For example: https://vCenter-host-example.com/sdk",
+  "Ensure the URL includes the \"/ovirt-engine/api\" path. For example: https://rhv-host-example.com/ovirt-engine/api.": "Ensure the URL includes the \"/ovirt-engine/api\" path. For example: https://rhv-host-example.com/ovirt-engine/api.",
```

Comment:
@RichardHoch based on https://developers.google.com/style/periods#periods-with-urls, it seems not recommended to put a period immediately after a URL. 
WDYT? Maybe adding quotes, i.e. `...For example: "https://rhv-host-example.com/ovirt-engine/api".?`